README.md

```diff
@@ -97,6 +97,20 @@ of asynchronous elements (for example parts loaded by AJAX).
 You can read more here:
 - [Browser Rendering](https://hexdocs.pm/crawly/basic_concepts.html#browser-rendering)
 
+## New: Experimental UI
+
+We have made few first steps towards building UI for Crawly project which would
```

Comment:
should be more concise.

The CrawlyUI project is an add-on that aims to provide an interface for managing and rapidly developing spiders.

Reply:
can replace the whole paragraph with this line. They say the same things

Reply:
Thanks!